web/src/user_group_edit.ts

```diff
@@ -799,7 +801,12 @@ export function update_group(event: UserGroupUpdateEvent): void {
     }
 
     if (event.data.description !== undefined) {
-        $group_row.find(".description").text(group.description);
+        const $edit_container = get_edit_container(group);
+        const html = render_user_group_description({
+            rendered_description: postprocess_content(group.rendered_description),
+        });
+        $edit_container.find(".group-description-wrapper").html(html);
+        $group_row.find(".description").html(html);
```

Comment:
Do we need to set the value rendered by `render_user_group_description` here? This is for the left panel of groups UI and I guess that is handled by `browse_user_groups_list_item` and it handles no description case using `data-no-description` attribute.